pkg/planner/core/casetest/cbotest/testdata/analyze_suite_xut.json

```diff
@@ -542,7 +542,7 @@
         "SQL": "EXPLAIN format=brief select /*+ nth_plan(5) */ * from `t19f3e4f1`  where  `colc864` in ( select  `colc864`     from  `t19f3e4f1`  where       `colaadb` in (      select     `colf2af`\n from   `t0da79f8d`  where  not ( `t19f3e4f1`.`colc864` <> null )  )  ) limit 2837;",
         "Plan": [
           "Limit 2.00 root  offset:0, count:2837",
-          "└─HashJoin 2.00 root  inner join, equal:[eq(test.t19f3e4f1.colc864, test.t19f3e4f1.colc864)]",
+          "└─IndexHashJoin 2.00 root  inner join, inner:IndexLookUp, outer key:test.t19f3e4f1.colc864, inner key:test.t19f3e4f1.colc864, equal cond:eq(test.t19f3e4f1.colc864, test.t19f3e4f1.colc864)",
```

Comment:
nice coverage, any idea about why this changes (which is different with the old one?)

Reply:
> nice coverage, any idea about why this changes (which is different with the old one?)

When Cascades is enabled, nth_plan hint will be [disabled](https://github.com/pingcap/tidb/blob/master/pkg/planner/core/find_best_task.go#L427-L428), and it will no longer step into [rebuildChildTasks](https://github.com/pingcap/tidb/blob/master/pkg/planner/core/find_best_task.go#L310-L318).